templates/openstack/managed.html

```diff
@@ -70,12 +70,12 @@ <h3 class="p-heading--four">Efficiency</h3>
     </div>
     <div class="col-4 p-divider__block">
       <h3 class="p-heading--four">Resilience</h3>
-      <p>High availability and resilience in storage and networking underpins our BootStack SLA and your peace of mind. Designed to be tough in the face of reality.</p>
+      <p>High availability and resilience in storage and networking underpins our managed services SLA and your peace of mind. Designed to be tough in the face of reality.</p>
```

Comment:
Slight difference in copy doc:
```suggestion
      <p>High availability and resilience in storage and networking underpins our managed services' SLA and your peace of mind. Designed to be tough in the face of reality.</p>
```